lib/Language/control.pod

```diff
@@ -238,6 +238,32 @@ two differences C<unless> works the same as L<if>:
 Implementation note: Currently, Rakudo will say "1 Nil 3 0" for the last
 example because it is not caught up to this part of the design yet.
 
+=head3 with, orwith, without
+
+The C<with> statement is like C<if> but tests for definedness rather than
```

Comment:
I don't think so, unless we have something to link to that's a description of definedness. I don't think it should be a link to C<defined> if that's what you're thinking.